packages/server/src/games.ts

```diff
@@ -86,20 +40,17 @@ function calculatePlayerAction(oldState: GameState, playerAction: PlayerAction)
       },
     ];
     newState.table.push(fromCardKey(playerAction.card));
-  } else if (playerAction.action === PlayerActionType.Capture) {
+  } else if (playerAction.action === PlayerActionType.Capture && playerAction?.tableCards !== undefined) {
```

Comment:
Can the `tableCards` be undefined if it's a capture?

Reply:
It shouldn't but I think it's for ts inference actually, we might make better types but it will be more complex

Reply:
```

export type PlayerAction = {
  action: PlayerActionType;
  playerName: string;
  card: string;
  tableCards?: string[];
};
```

Reply:
fixed